### DISCUSSION TITLE
Multiple harvester conundurm

### DISCUSSION THREAD

**Discussion by nomadengineer** [Chia-Network/chia-blockchain#3850] [General]
I implemented the multi-harvester network with 1 main node with all services on windows and 3 harvesters making plots on ubuntu.
This seemed like a nice idea as it would free me of moving the plots to the main farmer server.
HOWEVER what I noticed in the "Last Attempted Proof" section in the GUI that each harvester was getting a different challenge which it was evaluating on its farm of plots. This means that all harvesters are effectively sub-farms and a challenge would not necessarily be prooved against ALL plots that I have in all of my harvester. 
In my opinion this reduces my chances of winning, no?

Also with a lot of nodes connected sometimes I am getting a wide variety of block challenges and sometimes its just the same signature repeated 64 times and prooved for all harvesters.

Any explanation would be welcome! :)

Thank you.

**Comment by nomadengineer** [comment]
I am unable to find an answer to this anywhere! 
Please help me understand this! :(

Thank you!

**Reply by justwjx** [reply]
the remote harvester is quiet wired from log view now, so I have to just farm all plots with all plots mounted in one machine.

**Reply by justwjx** [reply]
I mean from full node / farm log view, I can not confirm it is working properly..

**Reply by nomadengineer** [reply]
From my harvester log view I can see that its getting challenges and sometimes pass a filter as well.
However what I am worried about it is that if it is getting ALL the challenges!
Also if my farmer is receiving enough challenges as well.

**Comment by anderzheng** [comment]
I think, the fullnode will pass the same challenges to multi-harvester.
I can see the same challenges at the GUI "Last Attempted Proof".
Maybe you can comfire the harvester's log, to see  if there are the same hash code in each.
![image](https://user-images.githubusercontent.com/23466259/117416810-55fdbe00-af4c-11eb-921c-d237dbaefde3.png)

**Reply by AL-XF** [reply]
I had the same experince like you since 3 weeks ago. My sub harvester always show 0/0 and same challenge as the Last attempted proof.
No clue what's difference between single harvester and multi-harvester.
